buildSrc/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +0,0 @@
-distributionBase=GRADLE_USER_HOME
```

Comment:
❌ I guess this `doLast` should also be removed: 
https://github.com/gradle/gradle/blob/c6c324e265c71f17fac79de32a01aaa7c824aa99/gradle/wrapper.gradle#L48-L55

Reply:
Done